tbx/core/blocks.py

```diff
@@ -162,6 +162,18 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/video_block.html"
 
 
+class PartnersBlock(blocks.StructBlock):
+    title = blocks.CharBlock(
+        max_length=255,
+    )
+    partner_logos = blocks.ListBlock(ImageChooserBlock(), label="Logos")
```

Comment:
I have realised we will probably need to editor to be able to set the alt text for each logo

Reply:
Decision made in standup to not include alt-text in MVP, will resolve thread